script.js

```diff
@@ -1,4 +1,400 @@
+      } points.`;
+    }
+    gameMode = "wager";
+    playerCounter = 0;
+    return `All of the player's name are inputted. Proceeding to getting the wagers.`;
+  }
+
+  if (gameMode == "wager") {
+    if (playerCounter < numberOfPlayers) {
+      playersObj[playerCounter].wager = Number(input);
+      playerCounter += 1;
+      return `Phew! you waged ${
+        playersObj[playerCounter - 1].wager
+      } points, let's see if you will win or lose that points.`;
+    }
+    gameMode = "start";
+    playerCounter = 0;
+    return `All of the player's wagers are inputted. Proceeding to the game.`;
+  }
+
+  if (gameMode == "start") {
+    var outputMessage = "Blackjack game is now starting, drawing cards.";
+    drawCards();
+    gameMode = "player";
+    return `${outputMessage}<br>Proceeding to ${playersObj[playerCounter].name} turn.`;
+  }
+
+  if (gameMode == "player") {
+    if (checkIfAllStand()) {
+      var outputMessage = drawBotHand();
+      gameState = true;
+      return `${outputMessage} <br><br>Click 'Submit' to continue.`;
+    }
+
+    var outputMessage = `This is ${playersObj[playerCounter].name}'s turn.<br><br>`;
+    if (!turn && playersObj[playerCounter].stand == false) {
+      outputMessage += `Player Hand<br>${displayHand(
+        playersObj[playerCounter].hand[0]
+      )}<br><br>Bot Hand<br>${displayHand(
+        botHand
+      )}<br><br>Do you want to 'hit' or 'stand'`;
+      if (
+        playersObj[playerCounter].hand[0][0].rank ==
+        playersObj[playerCounter].hand[0][1].rank
+      )
+        outputMessage += `or 'split'?`;
+      else {
+        outputMessage += `?<br>`;
+      }
+      turn = 1;
+      return outputMessage;
+    } else {
+      if (input == "stand") {
+        playersObj[playerCounter].stand = true;
+        outputMessage = `You chose to stand, please click "Submit" to continue.`;
+        playerCounter += 1;
+        turn = 0;
+        return outputMessage;
+      }
+      if (input == "hit") {
+        outputMessage += playBlackJackPlayer(
+          playersObj[playerCounter].hand[0],
+          input
+        );
+        turn = 0;
+        return outputMessage;
+      }
+      if (input == "split") {
+        playersObj[playerCounter].stand = true;
+        var firstCard = drawOneCard();
+        var secondCard = drawOneCard();
+        console.log(firstCard);
+        console.log(secondCard);
+        playersObj[playerCounter].hand.push([
+          playersObj[playerCounter].hand[0][1],
+          secondCard,
+        ]);
+        playersObj[playerCounter].hand[0].splice(1);
+        playersObj[playerCounter].hand[0].push(firstCard);
+        var outputMessage = `First Hand: <br>${displayHand(
+          playersObj[playerCounter].hand[0]
+        )}<br><br>Second Hand: <br>${displayHand(
+          playersObj[playerCounter].hand[1]
+        )} `;
+        turn = 0;
+        playerCounter += 1;
+        return outputMessage;
+      }
+    }
+  }
```

Comment:
great job in using helper function in your code! However, I am thinking that perhaps you can also further break down your code in the main function as currently, it spans over a hundred lines. Perhaps logic such as when the user input split could be broken down to smaller helper functions.